run_tests.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
This file seems like a script you could use locally to run using docker. I don't think it should be part of a PR.